src/backend/AbsSecBlock.c

```diff
@@ -55,24 +55,24 @@ bool BSL_AbsSecBlock_IsConsistent(const BSL_AbsSecBlock_t *self)
 void BSL_AbsSecBlock_Print(const BSL_AbsSecBlock_t *self)
 {
     BSL_StaticString_t str;
-    BSL_LOG_INFO("ASB  context id: %lu", self->sec_context_id);
+    BSL_LOG_INFO("ASB  context id: %" PRIu64, self->sec_context_id);
```

Comment:
The security context IDs should actually use a signed integer for storage. The valid range is 16-bit so okay to use int32_t or int64_t for consistency elsewhere. The format string would use a "d" instead of "u".